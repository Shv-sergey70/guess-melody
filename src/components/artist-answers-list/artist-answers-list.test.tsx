import * as React from 'react';
import {configure, shallow} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import ArtistAnswersList from "./artist-answers-list";

configure({adapter: new Adapter()});

const questionAnswers = [
  {
    picture: `test/track2.mp3`,
    artist: `Billie Eilish`
  },
  {
    picture: `test/track1.mp3`,
    artist: `Imagine Dragons`
  },
  {
    picture: `test/track3.mp3`,
    artist: `Artur Latte`
  }
];

test(`ArtistAnswersList correctly renders`, () => {
  const tree = shallow(
      <ArtistAnswersList
        questionAnswers={questionAnswers}
        onArtistSelect={jest.fn()}
        screenIndex={2}
      />
  );

  expect(tree).toMatchSnapshot();
});
