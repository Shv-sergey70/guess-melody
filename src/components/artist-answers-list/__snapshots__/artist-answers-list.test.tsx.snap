// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistAnswersList correctly renders 1`] = `ShallowWrapper {}`;
